tests/vmemcache_test_mt.c

```diff
@@ -113,12 +124,36 @@ worker_thread_get(void *arg)
 	return NULL;
 }
 
+/*
+ * worker_thread_put_in_gets -- (internal) worker testing vmemcache_put()
+ */
+static void *
+worker_thread_put_in_gets(void *arg)
+{
+	struct context *ctx = arg;
+	unsigned long long i;
+	unsigned long long start = ctx->ops_count + (ctx->thread_number & 0x1);
+	unsigned long long end = 3 * ctx->ops_count;
```

Comment:
why 3? Let's make this a separate option.

Reply:
I added a comment here